src/StructuredLogViewer.Core/ProjectGraph/RuntimeGraph.cs

```diff
@@ -40,15 +43,37 @@ public RuntimeGraphNode(Project p)
                 Project = p;
             }
 
+            public string DebuggerDisplayString()
+            {
+                var sb = new StringBuilder();
+
+                sb.Append($"ReferenceCount: {SortedChildren.Count}, {Project.DebuggerDisplayString()}");
+
+                return sb.ToString();
+            }
+
             internal void AddChild(RuntimeGraphNode child)
             {
                 if (sortedChildren == null)
                 {
                     sortedChildren = new SortedList<DateTime, RuntimeGraphNode>();
                 }
 
-                sortedChildren.Add(child.Project.StartTime, child);
+                // some projects appear to have the same timestamp, add 1 tick to avoid a key already exists exception from the sorted list
+                var projectStartTime = GetNearestNonConflictingDateTime(child.Project.StartTime, sortedChildren);
+
+                sortedChildren.Add(projectStartTime, child);
                 sortedChildrenCached = null;
+
+                DateTime GetNearestNonConflictingDateTime(DateTime newKey, SortedList<DateTime, RuntimeGraphNode> collection)
+                {
+                    while (collection.ContainsKey(newKey))
+                    {
+                        newKey = new DateTime(newKey.Ticks + 1);

```

Comment:
this would preserve time zone info (not sure that's relevant here):

```suggestion
                        newKey = newKey.AddTicks(1);
```